src/main/kotlin/de/itemis/mps/gradle/tasks/backend_arguments.kt

```diff
@@ -10,14 +10,14 @@ import org.gradle.api.logging.LogLevel
 import org.gradle.api.provider.Provider
 import java.io.File
 
-internal fun Task.addLogLevel(args: MutableCollection<String>) = addIfInfoLogLevel(args, "--log-level=info")
+internal fun Task.addLogLevel(args: MutableCollection<String>, logLevel: LogLevel) = addIfInfoLogLevel(args, logLevel,"--log-level=info")
 
 /**
  * A weird overload, usable for building a map of Ant task attributes, as well as a list of command line backend
  * attributes.
  */
-internal fun <T> Task.addIfInfoLogLevel(result: MutableCollection<T>, value: T) {
-    val effectiveLogLevel = logging.level ?: project.gradle.startParameter.logLevel
+internal fun <T> Task.addIfInfoLogLevel(result: MutableCollection<T>, logLevel: LogLevel, value: T) {
```

Comment:
The two `add*` methods should no longer need `Task` as their receiver. This one could probably even be inlined because it doesn't bring much value anymore, apart from knowing that 'INFO' is somehow a significant log level, but I'm on the fence here.

Reply:
Ah right. But I thought we wanted to get rid of that anyway? I have never actually seen task's logging.level be different from null.

Reply:
True, I'll simplify it some more 👍